### PR TITLE
Add make target `delete-docker-image`

### PR DESCRIPTION
**Pull request by schu**
Otherwise we sometimes run out of disk space on e.g. semaphoreci.

### CODE REVIEW

File: environments/debian-testing.mk [hunk 1]
```diff
@@ -2,7 +2,7 @@ include build.mk
 
 DISTRO=debian-testing
 
-build: build-docker-image build-ebpf-object
+build: build-docker-image build-ebpf-object delete-docker-image delete-docker-image
```
Comment (alban) [inline environments/debian-testing.mk]: Target written twice

**Comment by alban** [comment]
Except typo above, lgtm if CI is green.

File: environments/debian-testing.mk (hunk 1 above)
Comment (schu) [inline environments/debian-testing.mk]: Oops, thanks.

**Pull request merged**